core/src/main/java/org/mule/util/OneTimeWarning.java

```diff
@@ -35,10 +35,17 @@ public void warn()
         delegate.warn();
     }
 
+    public void warn(Object... args)
```

Comment:
add javadocs to this method and the other overload so that the difference between them is clear to users